pop-up.css

```diff
@@ -0,0 +1,203 @@
+* {
+  margin: 0;
+  padding: 0;
+  outline: 0;
+  border: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  list-style: none;
+}
+
+html {
+  scroll-behavior: smooth;
+  overflow-x: hidden;
+}
+
+body {
+  background: #eee;
+}
+
+#modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  min-height: 100vh;
+  width: 100%;
+  background: rgba(0, 0, 0, 0.8);
```

Comment:
- [ ] Please note the modal `background-color` on the Figma design, it is different from your color. Kindly check it to follow the Figma design as requested in the project requirement.

### Figma Design :point_down: 
![Screenshot from 2023-04-12 15-24-10](https://user-images.githubusercontent.com/91142432/233095787-dade7525-7e0f-474d-896c-dd7a60c2f4ba.png)

### Your Design :point_down: 
![Screenshot from 2023-04-19 13-42-25](https://user-images.githubusercontent.com/91142432/233095981-7429464f-0541-420b-be5e-780db722ec60.png)

Reply:
Changed Background to match figma design